src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from "@chakra-ui/react"
+import { Story, ComponentMeta } from "@storybook/react"
+
+import { Footer } from "./Footer"
+
+const footerMeta = {
+  title: "Components/Footer",
+  component: Footer,
+} as ComponentMeta<typeof Footer>
+
+interface TemplateArgs {
+  buttonText: string
+}
+
+const SingleButtonFooterTemplate: Story<TemplateArgs> = ({
+  buttonText,
+}: TemplateArgs) => {
+  return (
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <Footer>
+      <Button>{buttonText}</Button>
+    </Footer>
+  )
+}
+
+interface MultiButtonFooterTemplateArgs extends TemplateArgs {
+  button2Text: string
+}
+const MultiButtonFooterTemplate: Story<MultiButtonFooterTemplateArgs> = ({
+  buttonText,
+  button2Text,
+}: MultiButtonFooterTemplateArgs) => {
+  return (
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <Footer>
+      <>
```

Comment:
we don't need to wrap in a fragment

Reply:
Right, I was having issues with this earlier because of the props in Footer - but swapping to `PropsWithChildren` fixed the issue! https://github.com/isomerpages/isomercms-frontend/pull/932/commits/7faa682fd7af5140926947afef3520d12daa67fb